versions

```diff
@@ -1,6 +1,6 @@
 # Last releases of all Actyx products
 # Each line contains <release> <commit-hash>
-# The machine-readable product names are: actyx, node-manager,
+# The machine-readable product names are: ax, ax_core, actyx, node-manager,
```

Comment:
`ax-types`?

Reply:
Doesn't exist in this PR yet and it's precisely the ones we need to release by hand

Reply:
Ah, nvm. I was confused about the purpose of this thing